src/com/team2502/robot2018/command/autonomous/groups/LeftCommandGroup.java

```diff
@@ -25,6 +26,7 @@ public LeftCommandGroup()
                     case SCALE:
                     {
                         goScaleLeft();
+                        //secondCubeLeft();
```

Comment:
TODO: Make second cube work

Reply:
I made it so the second Pure Pursuit command should rely on absolute coordinates (technically relative to first location in the match) due to there being substantial inconsistencies in where the robot ends up after one command... The Butterfly Effect is not good (also we should sometime try reverse Pure Pursuit).